website_media_size/static/src/js/website_media_size.js

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2018 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('website_media_size', function(require) {
+    var MediaDialog = require('web_editor.widget').MediaDialog;
+    var weContext = require("web_editor.context");
+
+    MediaDialog.include({
+        xmlDependencies: MediaDialog.prototype.xmlDependencies.concat(
+            ['/website_media_size/static/src/xml/website_media_size.xml']
+        ),
+        start: function() {
+            var res = this._super.apply(this, arguments);
+            this.include_file_size(this.imageDialog);
+            this.include_file_size(this.documentDialog);
+             // FIXME
```

Comment:
?

Reply:
The this.search(); is a weird fix to update the size values. I couldn't find a proper solution. There must be a better way to fix it.